src/index.js

```diff
@@ -4,21 +4,24 @@ exports = module.exports = Repo
 
 function Repo (repoPath, options) {
   this.init = (config, callback) => {
-    if (this.exists()) {
-      throw new Error('Repo already exists')
-    }
+    this.exists((err, exists) => {
+      if (err) { throw err }
+      if (exists) { throw new Error('Repo already exists') }
+      throw new Error('not implemented')
```

Comment:
There's no further implementation of `repo.init` yet, so 'not implemented' seemed fitting.